papers/adam_richie-halford/adam_richie-halford.rst

```diff
@@ -653,13 +612,16 @@ because they show potential users the relative cost in execution time of
 using Cloudknot in comparison to other distributed computing platforms.
 
 On the other hand, the scaling results in this paper, indeed most
-scaling results in general, measure :math:`t_\mathrm{exec}` from
-Eq :ref:`eq.tcompute`, capturing only partial information about
-:math:`t_\mathrm{compute}`. Precisely measuring :math:`t_\mathrm{env}`
-is beyond the scope of this paper so the reduction in
-:math:`t_\mathrm{compute}` is admittedly speculative. But we believe an
-increase in execution time may be acceptable in some situations. For
-example, if the amount of time that a user will spend learning a new
+scaling results in general, measure the bare execution time, capturing
+only partial information about the time that it takes to reach a
+computational result. This is because all the distributed systems
+currently available require some amount of systems administration and
+often incur non-trivial setup time. In addition, most of the existing
+systems currently require some amount of rewriting of the original code
+(see :cite:`mehta2017comparative` for further analysis). We believe an
```

Comment:
Reviewer #2 will likely ask to remove the parenthetical "(see [] for further analysis)" in favor of simply citing the reference after the sentence. He had a similar comment about another reference like that elsewhere in the paper.